sdk/evaluation/azure-ai-evaluation/tests/e2etests/test_mass_evaluate.py

```diff
@@ -0,0 +1,372 @@
+        assert len(row_result_df["outputs.indirect_attack.xpia_intrusion"]) == 2
+        assert len(row_result_df["outputs.indirect_attack.xpia_information_gathering"]) == 2
+        assert len(row_result_df["outputs.indirect_attack.evaluation_per_turn"]) == 2
+        assert len(row_result_df["outputs.eci.eci_label"]) == 2
+        assert len(row_result_df["outputs.eci.evaluation_per_turn"]) == 2
+        assert len(row_result_df["outputs.content_safety.sexual_score"]) == 2
+        assert len(row_result_df["outputs.content_safety.violence_score"]) == 2
+        assert len(row_result_df["outputs.content_safety.self_harm_score"]) == 2
+        assert len(row_result_df["outputs.content_safety.hate_unfairness_score"]) == 2
+        assert len(row_result_df["outputs.content_safety.evaluation_per_turn"]) == 2
+        assert len(row_result_df["outputs.retrieval.retrieval"]) == 2
+        assert len(row_result_df["outputs.retrieval.gpt_retrieval"]) == 2
+        assert len(row_result_df["outputs.retrieval.evaluation_per_turn"]) == 2
+
+        assert len(metrics.keys()) == 21
+        assert metrics["coherence.coherence"] >= 0
+        assert metrics["coherence.gpt_coherence"] >= 0
+        assert metrics["fluency.fluency"] >= 0
+        assert metrics["fluency.gpt_fluency"] >= 0
+        assert metrics["relevance.relevance"] >= 0
+        assert metrics["relevance.gpt_relevance"] >= 0
+        assert metrics["grounded.gpt_groundedness"] >= 0
+        assert metrics["grounded.groundedness"] >= 0
+        assert metrics["retrieval.retrieval"] >= 0
+        assert metrics["retrieval.gpt_retrieval"] >= 0
+        assert metrics["indirect_attack.xpia_manipulated_content"] >= 0
+        assert metrics["indirect_attack.xpia_intrusion"] >= 0
+        assert metrics["indirect_attack.xpia_information_gathering"] >= 0
+        assert metrics["content_safety.sexual_defect_rate"] >= 0
+        assert metrics["content_safety.violence_defect_rate"] >= 0
+        assert metrics["content_safety.hate_unfairness_defect_rate"] >= 0
+        assert metrics["content_safety.self_harm_defect_rate"] >= 0
+        assert metrics["grounded_pro.groundedness_pro_passing_rate"] >= 0
+        assert metrics["protected_material.protected_material_defect_rate"] >= 0
+        assert metrics["indirect_attack.xpia_defect_rate"] >= 0
+        assert metrics["eci.eci_defect_rate"] >= 0
+
+    # Imagee urls with target is disabled due to being unstable in CI
+    @pytest.mark.parametrize(
+        "multi_modal_input_type",
+        [
+            "imageurls",
+            # "imageurls_with_target",
+            "b64_images",
+        ],
+    )
+    def test_evaluate_multimodal(self, multi_modal_input_type, multimodal_input_selector, azure_cred, project_scope):
+        # Content safety is removed due to being unstable in playback mode
+        evaluators = {
+            # "content_safety" : ContentSafetyMultimodalEvaluator(credential=azure_cred, azure_ai_project=project_scope),
```

Comment:
What happens when you enable it. Does it fail the test or skip it?
do you have any theory or know root cause, why It must be happening?